wheel/src/api.rs

```diff
@@ -365,6 +371,47 @@ fn fast_forward_singleton<'p>(
     ))
 }
 
+#[pyfunction]
+#[pyo3(name = "validate_clvm_and_signature")]
+pub fn py_validate_clvm_and_signature(
+    new_spend: SpendBundle,
+    max_cost: u64,
+    constants: ConsensusConstants,
+    peak_height: u32,
+    cache: BlsCache,
+) -> PyResult<(
+    SpendBundle,
+    OwnedSpendBundleConditions,
+    HashMap<[u8; 32], GTElement>,
```

Comment:
I think it would be preferable to return a `Vec<>` here, rather than a hashmap. All we're doing with it is pickling, unpickling and adding it to the main BLS cache